### DISCUSSION TITLE
bug: Font not working

### DISCUSSION THREAD

**Discussion by Biki-das** [tailwindlabs/tailwindcss#11114] [Help]
**What version of Tailwind CSS are you using?**
3.3.1

**What build tool (or framework if it abstracts the build tool) are you using?**

For example: vite postcss 3.3.1

**What version of Node.js are you using?**

For example: v12.0.0

**What browser are you using?**

For example: Chrome, Safari, or N/A

**What operating system are you using?**

For example: macOS, Windows

**Reproduction URL**

A Tailwind Play link or public GitHub repo that includes a minimal reproduction of the bug. **Please do not link to your actual project**, what we need instead is a _minimal_ reproduction in a fresh project without any unnecessary code. This means it doesn't matter if your real project is private/confidential, since we want a link to a separate, isolated reproduction anyways.

A reproduction is **required** when filing an issue — any issue opened without a reproduction will be closed and you'll be asked to create a new issue that includes a reproduction. We're a small team and we can't keep up with the volume of issues we receive if we need to reproduce each issue from scratch ourselves.
A Tailwind Play link or public GitHub repo that includes a minimal reproduction of the bug. **Please do not link to your actual project**, what we need instead is a _minimal_ reproduction in a fresh project without any unnecessary code. This means it doesn't matter if your real project is private/confidential, since we want a link to a separate, isolated reproduction anyways.

A reproduction is **required** when filing an issue — any issue opened without a reproduction will be closed and you'll be asked to create a new issue that includes a reproduction. We're a small team and we can't keep up with the volume of issues we receive if we need to reproduce each issue from scratch ourselves.

**Describe your issue**

I am using a set of font families which one of them gets applied, for some reason when i do something like font-${xxxxx} , this does not apply the font but just doing this like font-${"xxxx"} works?!! also on restarting the server the font families do not load, they only load after i write font-xxxx

here is some code isn't this valid to do in tailwind
```const colors = {
  Orange: "#f87070",
  Teal: "#70F3F8",
  Purple: "#D881F8",
};

const colorTypes = ["Orange", "Teal", "Purple"];

function ThemeSelector({ currentFont }) {
  return (
    <div className="mx-auto  mt-4 flex w-[90%] flex-col items-center justify-between md:flex-row">
      <p
        className={`text-center font-${currentFont} text-sm font-bold tracking-widest`}
      >
        C O L O R
      </p>
      <div className="mt-4 flex gap-x-6">
        {colorTypes.map((color) => {
          return (
            <div key={color} className={`bg-[${colors[color]}]`}>
              {color}
            </div>
          );
        })}
      </div>
    </div>
  );
}

export default ThemeSelector;

**The problem above is only the orange color renders rest color do not appear in the background!**



```

**Comment by Steffan153** [comment]
Tailwind doesn't support dynamic classes. It is based on your actual React code, not your props. See https://stackoverflow.com/q/69687530
